code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -0,0 +1,40 @@
+/obj/item/stack/nanopaste
+	name = "nanopaste"
+	singular_name = "nanite swarm"
+	desc = "A tube of paste containing swarms of repair nanites. Very effective in repairing robotic machinery."
+	icon = 'icons/obj/nanopaste.dmi'
+	icon_state = "tube"
+	origin_tech = "materials=2;engineering=3"
+	amount = 10
+	toolspeed = 1
+
+/obj/item/stack/nanopaste/attack(mob/living/M as mob, mob/user as mob)
+	if(!istype(M) || !istype(user))
+		return 0
+	if(istype(M,/mob/living/silicon/robot))	//Repairing cyborgs
+		var/mob/living/silicon/robot/R = M
+		if(R.getBruteLoss() || R.getFireLoss() )
+			R.adjustBruteLoss(-15)
+			R.adjustFireLoss(-15)
+			R.updatehealth()
+			use(1)
+			user.visible_message("<span class='notice'>\The [user] applied some [src] at [R]'s damaged areas.</span>",\
+				"<span class='notice'>You apply some [src] at [R]'s damaged areas.</span>")
+		else
+			to_chat(user, "<span class='notice'>All [R]'s systems are nominal.</span>")
+
+	if(istype(M,/mob/living/carbon/human))		//Repairing robolimbs
+		var/mob/living/carbon/human/H = M
+		var/obj/item/bodypart/BP = H.get_bodypart(user.zone_selected)
```

Comment:
The biggest potential problem is this not accounting for organs. I have no clue if it does or not.

Reply:
This does not account for organs, but those can't get damaged atm. Still waiting for my medical rework which will come soon™.

For now this is fine.